client/src/components/Availability/DesktopAvailability.tsx

```diff
@@ -0,0 +1,74 @@
+import { Box, CircularProgress, Grid, Link } from '@mui/material';
+import { makeStyles } from '@mui/styles';
+import { cloneElement } from 'react';
+import { NavLink, Redirect, Route, Switch, useHistory } from 'react-router-dom';
+import PageContainer from '../../components/PageContainer/PageContainer';
+import SettingHeader from '../../components/SettingsHeader/SettingsHeader';
+import SettingsWrapper from '../../components/SettingsWrapper/SettingsWrapper';
+import { useAuth } from '../../context/useAuthContext';
+import EditProfile from '../../pages/Settings/EditProfile/EditProfile';
+import Availability from '../../pages/Settings/Availability/Availability';
+
+const useStyles = makeStyles({
+  activeLink: {
+    fontWeight: 700,
+    color: '#000',
+  },
+});
+
+const DesktopAvailability = (): JSX.Element => {
+  const settingsMenu = [
+    {
+      name: 'Edit profile',
+      to: '/profile/settings/edit-profile',
+      component: <EditProfile header="Edit Profile" />,
+    },
```

Comment:
This isn't at all what we should be doing, there's already a Settings page that exists. You just need to create the page for the availability and set it up in the settings menu list in that component. Please remove this component.